pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -23,58 +23,61 @@ import (
 )
 
 var (
-	// The predicateName tries to be consistent as the predicate name used in DefaultAlgorithmProvider defined in
-	// defaults.go (which tend to be stable for backward compatibility)
+	// The first argument of newPredicateFailureError is predicate name, which
```

Comment:
Now that you have changed them to constants, looks like we don't need this comment anymore?

Reply:
OK, thx